src/enums/enum.ts

```
@@ -71,3 +71,25 @@ export declare enum CustomHandshakeProtocol {
+}
+
+export enum RESULT {
+  SUCCESS = 'SUCCESS',
+  FAILED = 'FAILED',
+}
+
+export enum SCOPES {
+  UNPROTECTED = 'skip',
+  MULTITENANT_BASE_AGENT = 'Basewallet',
+  TENANT_AGENT = 'tenant',
+  DEDICATED_AGENT = 'dedicated'
```

ESLint

[error] Insert `,`

(prettier/prettier)